Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/ConnectionManagerBean.java

```diff
@@ -95,66 +96,38 @@ public void setSelectedEntityName(String selectedEntityName) {
         }
     }
 
-    public BusinessEntity getSelectedEntity() {
+    public OrganizationType getSelectedEntity() {
         return selectedEntity;
     }
 
     public String getSelectedEntityDescription() {
         String description = NULL_DISPLAY;
-
-        if (selectedEntity != null && selectedEntity.getDescription() != null
-            && !selectedEntity.getDescription().isEmpty()) {
-            description = selectedEntity.getDescription().get(0).getValue();
-        }
-
+        // OrganizationType no longer have Description
```

Comment:
this method only return nulldisplay. can we just remove it then

Reply:
Need to decide if we want description in new exchange schema.